prime-router/start_func.sh

```diff
@@ -23,4 +23,10 @@ fi
 
 cd $function_folder
 
+# Use the prime CLI to load configuration in the background
+# The fatjar is two levels up in the libs folder
+top_dir=$function_folder/../..
+fatjar=$top_dir/libs/prime-router-0.1-SNAPSHOT-all.jar
+java -jar $fatjar lookuptables loadall -d $top_dir/resources/test/metadata/tables -r 10 | awk '{print "[LOAD TABLES] " $0}' &
```

Comment:
Some notes here:
1. I use awk to add text, so it is easy to see the outout.
2. I am aiming to the same thing here for the settings table.  I would like to remove the use of the settings container to save memory.  Thoughts?

Reply:
Check on loading the tables only when they change.

Reply:
Agreed - it would be great if this did not load on every docker-up unless changes are present.